draft-ietf-quic-bit-grease.md

```diff
@@ -94,13 +94,17 @@ assigns specific meaning to the value of the bit.
 All packets sent after receiving and processing transport parameters might be
 affected, including Initial, Handshake, and Retry packets.
 
-A client MAY also clear the QUIC Bit in Initial packets that are sent prior to
-receiving transport parameters from the server.  A client can only clear the
-QUIC Bit if such packets include a token provided by the server in a NEW_TOKEN
-frame ({{Section 19.7 of QUIC}}) on a connection where the server also included
-the grease_quic_bit transport parameter.  To allow for changes in server
-configuration, clients SHOULD set the QUIC Bit if the token was provided more
-than 7 days prior.
+A client MAY also clear the QUIC Bit in packets that are sent prior to receiving
+transport parameters from the server.  However, a client MUST NOT clear the QUIC
+Bit unless the Initial packets it sends include a token provided by the server
+in a NEW_TOKEN frame ({{Section 19.7 of QUIC}}) received less than 7 days ago on
```

Comment:
nit: nit a huge fan of "7 days ago", the older text was more precise

Reply:
On further thought,  is elapsed time in days or calendar days? Maybe you want to use hours for precision and state this is 7 days for reader affordable.

Reply:
how about
```suggestion
in a NEW_TOKEN frame ({{Section 19.7 of QUIC}}) received less than 604800 seconds (7 days) ago on
```

Reply:
or maybe even
```suggestion
in a NEW_TOKEN frame ({{Section 19.7 of QUIC}}) received less than 604800 seconds (7 days) prior on
```